components/Nav.js

```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link'
+import styles from "../styles/Nav.module.css"
+import Button from "./Button"
+
+const Nav = () => {
+  return (
+   <> 
+    <nav className={styles.nav}>
+      <ul className={styles.Agenda}>
+        <li>
+          <Link href='/home'>Agenda</Link>
```

Comment:
```suggestion
          <Link href='/'>Agenda</Link>
```
Just to fix the link for now.